src/ru/skypro/Car.java

```diff
@@ -0,0 +1,9 @@
+package ru.skypro;
+
+
+public class Car extends MotorTransport{
+
+    public Car(String modelName) {
+        super(modelName,4);
```

Comment:
```suggestion
        super(modelName, 4);
```
Можете использовать ctrl+alt+L для автоформатирования